crates/sui-protocol-config/src/lib.rs

```diff
@@ -884,6 +885,11 @@ pub struct ProtocolConfig {
     /// Maximum allowed precision loss when reducing voting weights for the random beacon
     /// protocol.
     random_beacon_reduction_allowed_delta: Option<u16>,
+
+    /// The maximum serialised transaction size (in bytes) accepted by consensus
+    consensus_max_transaction_size_bytes: Option<u64>,
+    /// The maximum size of transactions included in a consensus proposed block
+    consensus_max_block_transactions_size_bytes: Option<u64>,
```

Comment:
Totally optional: maybe `consensus_max_transactions_in_block_size_bytes`? cc @arun-koshy

Reply:
I like `consensus_max_transactions_in_block_size_bytes`! Only thing I would add is that `bytes` implies size right? So maybe we can just have the fields end in `bytes` instead of `size_bytes`? But another optional option on top of mingwei's optional option 😄

Reply:
> So maybe we can just have the fields end in bytes instead of size_bytes

Good point, yes I think we can do that 👍 

> But another optional option on top of mingwei's optional option

ha oh that reminds me [Inception](https://www.imdb.com/title/tt1375666/)